pom.xml

```diff
@@ -545,6 +545,42 @@
 
     <build>
         <plugins>
+            <plugin>
+                <artifactId>exec-maven-plugin</artifactId>
+                <groupId>org.codehaus.mojo</groupId>
+                <executions>
+                    <execution>
+                        <id>Generate sources</id>
```

Comment:
We can remove the ID since there is only one execution

Reply:
Removed.